sdlib/d/gc/base.d

```diff
@@ -86,6 +86,18 @@ public:
 		return (cast(Base*) &this).reserveAddressSpaceImpl(size, alignment);
 	}
 
+	void purgeAddressSpace(void* address, size_t size) shared {
```

Comment:
Why is this in base? it's using none of the data from the base, so that doesn't seems necessary. In addition, why is this taking the lock?

Reply:
It is not using anything in base. It's not about where I want it, it's about where it belongs. You need to be finding these thing out or we are condemned to have endless back and forth.

Reply:
What is purgeAdressSpace for here?

Reply:
Why is it in the base?